doc/screen-recording.md

```diff
@@ -0,0 +1,69 @@
+# Genesys Cloud WebRTC SDK Screen Recording
+
+This SDK supports abstracting the screen recording process that occurs inside the Genesys Cloud Desktop App. Screen 
+recordings are initiated by the server which notifies the webrtc sdk who then abstracts nearly all of the setup and
+signaling to make the screen recording work.
+
+> *Note: screen recording does not support guest users.*
+
+## Prerequisites
+
+You must have screen recording policies in place and the Webrtc Sdk must be configured to allow screen recording sessions.
```

Comment:
Just reading this over once, maybe we should add a little bit more information about _how_ screen recording works. In these examples, we `await sdk.startVideoConference(roomJid);` but it isn't super clear how/if that is what triggers the screen recording. 

Granted, most developers using the SDK for recording _should_ already have a knowledge of how SR works. But I know we are changing how it works. Thinking something along the lines of: 

> When the server determines a conversation should be recorded, it will send a pendingSession similar to how other session types work (ie. `softphone` and `video` conversations). It will be the responsibility of the consuming client to accept the pending session, gather media, accept the session, and add the media to that session.